### PR TITLE
pw 1867 - local payment methods

### PR DESCRIPTION
**Pull request by KadoBOT**
Adds local payment methods feature

### CODE REVIEW

File: packages/client-extension/widget/AdyenPaymentWidget/js/constants/index.js [hunk 1]
```diff
@@ -8,17 +8,17 @@ export const errorMessages = {
 export const paymentMethodTypes = {
     generic: 'generic',
     invoice: 'invoice',
+    local: 'local',
 }
-export const adyenCheckoutComponentUrl =
-    'https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.3.0/adyen.js'
+export const adyenCheckoutComponentUrl = 'https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.3.0/adyen.js'
```
Comment (rikterbeek) [inline packages/client-extension/widget/AdyenPaymentWidget/js/constants/index.js]: make it dynamic based on mode. TEST/LIVE

File: packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/boleto.spec.js.snap [hunk 2]
```diff
@@ -0,0 +1,98 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Boleto should display boleto component 1`] = `
+Object {
+  Symbol(dom): <body>
+    <link
+      href="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.3.0/adyen.css"
+      rel="stylesheet"
```
Comment (rikterbeek) [inline packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/boleto.spec.js.snap]: same here

File: packages/client-extension/widget/AdyenPaymentWidget/templates/display.template [hunk 3]
```diff
@@ -1,40 +1,20 @@
-<link rel="stylesheet" href="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.2.0/adyen.css" />
+<link rel="stylesheet" href="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.3.0/adyen.css" />
```
Comment (rikterbeek) [inline packages/client-extension/widget/AdyenPaymentWidget/templates/display.template]: make it dynamic based on mode. TEST/LIVE

File: packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/card.spec.js.snap [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+  </body>,
+}
+`;
+
+exports[`Card should display installments 1`] = `
+Object {
+  Symbol(dom): <body>
+    <link
+      href="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.3.0/adyen.css"
+      rel="stylesheet"
```
Comment (rikterbeek) [inline packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/card.spec.js.snap]: same here

File: packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/local.spec.js.snap [hunk 5]
```diff
@@ -0,0 +1,264 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Local should render 1`] = `
+Object {
+  Symbol(dom): <body>
+    <link
+      href="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.3.0/adyen.css"
+      rel="stylesheet"
+    />
```
Comment (rikterbeek) [inline packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/local.spec.js.snap]: same here

File: packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/card.spec.js.snap [hunk 6]
```diff
@@ -0,0 +1,198 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Card should display combo card 1`] = `
+Object {
+  Symbol(dom): <body>
+    <link
+      href="https://checkoutshopper-live.adyen.com/checkoutshopper/sdk/3.3.0/adyen.css"
+      rel="stylesheet"
```
Comment (rikterbeek) [inline packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/card.spec.js.snap]: same here

File: packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/template.spec.js.snap [hunk 7]
```diff
@@ -28,21 +29,37 @@ Object {
...
+        <div
+          class="loader-wrapper hide"
+        >
+          <div
+            class="lds-ring"
+          >
+            <div />
+            <div />
+            <div />
+            <div />
```
Comment (msilvagarcia) [inline packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/template.spec.js.snap]: Would be nice to have a comment explaining these 4 empty divs in a row

File: packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/template.spec.js.snap [hunk 8]
```diff
@@ -171,8 +222,25 @@ Object {
       </div>
       <div
         class="adyen-wrapper collapse"
-        id="adyen-card-wrapper"
+        id="adyen-generic-wrapper"
```
Comment (msilvagarcia) [inline packages/client-extension/widget/AdyenPaymentWidget/js/__tests__/__snapshots__/template.spec.js.snap]: I see this is a snapshot, but it looks like this id is duplicated.

File: packages/client-extension/widget/AdyenPaymentWidget/js/components/local/index.js [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+          <button>${name}</button>
+        </span>
+      </strong>`
+    const component = document.createElement('div')
+    component.setAttribute('id', `adyen-${type}-wrapper`)
+    component.classList.add('adyen-wrapper', 'collapse')
+
+    component.innerHTML = `
+        <div class="loader-wrapper hide">
+            <div class="lds-ring"><div></div><div></div><div></div><div></div></div>
```
Comment (msilvagarcia) [inline packages/client-extension/widget/AdyenPaymentWidget/js/components/local/index.js]: Would be nice to have a comment explaining this many empty divs

**Pull request merged**